package.json

```diff
@@ -10,6 +10,7 @@
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-router-dom": "^6.14.1",
     "react-scripts": "5.0.1",
     "web-vitals": "^2.1.4"
```

Comment:
- [ ] The app works as expected, but there is an error that shows up on the screen. Kindly fix this issue and provide enough information required for running your project in the readme file.

![error](https://github.com/supreeti/maths-magician-app/assets/53124548/dd1893ed-1f89-4ead-8c89-57e8f0232b23)